src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -315,8 +315,8 @@ def load_arguments(self, _):
         c.argument('auto_upgrade_channel', arg_type=get_enum_type(auto_upgrade_channels))
         c.argument('api_server_authorized_ip_ranges',
                    type=str, validator=validate_ip_ranges)
-        c.argument('enable_ahub', options_list=['--enable-ahub'])
-        c.argument('disable_ahub', options_list=['--disable-ahub'])
+        c.argument('enable_ahub', options_list=['--enable-ahub'], action='store_true')
+        c.argument('disable_ahub', options_list=['--disable-ahub'], action='store_true')
```

Comment:
Will the change of parameter type cause breaking change to users?

Reply:
By default, the option will be parsed as an ordinary parameter (which should take a string-like argument), and before it was implicitly inferred by the cli through the default value of the parameter in the function of the bool type. Now I am adding action='store_true' to the these places to express this clearly.